main/manager_doc.xml

```diff
@@ -0,0 +1,1623 @@
+						<para>
+						Only events with name <replaceable>event_name</replaceable> will be included.
+						<replaceable>event_name</replaceable> may be prefixed with an exclamation point
+						to exclude events with that name.
+						</para>
+						<para>
+						</para>
+					</enum>
+					<enum name="header(&lt;header_name&gt;)">
+						<para>
+						Only events containing a header with a name of <replaceable>header_name</replaceable>
+						will be included and the <literal>Filter</literal> parameter (if supplied) will only be
+						matched against the value of the header.
+						</para>
+						<para>
+						</para>
+					</enum>
+					<enum name="&lt;match_method&gt;">
+						<para>Specifies how the <literal>Filter</literal> parameter
+						is to be applied to the results of applying any 
+						<literal>name(&lt;event_name&gt;)</literal> and/or
+						<literal>header(&lt;header_name&gt;)</literal> parameters
+						above.
+						</para>
+						<para>
+						One of the following:
+						</para>
+						<enumlist>
+							<enum name="regex">
+								<para>The <literal>Filter</literal> parameter contains a regular expression
+								which will be matched against the result. (default)
+								</para>
+								<para>
+								</para>
+							</enum>
+							<enum name="exact">
+								<para>The <literal>Filter</literal> parameter contains a string which will
+								be exactly matched to the result.
+								</para>
+								<para>
+								</para>
+							</enum>
+							<enum name="contains">
+								<para>The <literal>Filter</literal> parameter contains a string
+								which will be searched for in the result.
+								</para>
+								<para>
+								</para>
+							</enum>
+							<enum name="none">
+								<para>The <literal>Filter</literal> parameter is ignored.
+								</para>
+							</enum>
+						</enumlist>
+					</enum>	
+				</enumlist>
+			</parameter>
+			<parameter name="Filter">
+				<para>Filters can be whitelist or blacklist</para>
+				<para>Example whitelist filter: "Event: Newchannel"</para>
+				<para>Example blacklist filter: "!Channel: DAHDI.*"</para>
+				<para>This filter option is used to whitelist or blacklist events per user to be
+				reported with regular expressions and are allowed if both the regex matches
+				and the user has read access as defined in manager.conf. Filters are assumed to be for whitelisting
+				unless preceeded by an exclamation point, which marks it as being black.
+				Evaluation of the filters is as follows:</para>
+				<para>- If no filters are configured all events are reported as normal.</para>
+				<para>- If there are white filters only: implied black all filter processed first, then white filters.</para>
+				<para>- If there are black filters only: implied white all filter processed first, then black filters.</para>
+				<para>- If there are both white and black filters: implied black all filter processed first, then white
+				filters, and lastly black filters.</para>
```

Comment:
include/exclude list

Reply:
I removed this section entirely and replaced it with...

> See the manager.conf.sample file in the configs/samples directory of the Asterisk source tree for a full description and examples.